classes/class-wc-connect-shipping-label.php

```diff
@@ -411,13 +411,14 @@ protected function reducer_items_quantity( $sum, $item ) {
 		}
 
 		public function meta_box( $post, $args ) {
+			$connect_order_presenter = new WC_Connect_Order_Presenter();
+
 			$order = wc_get_order( $post );
 			$order_id = WC_Connect_Compatibility::instance()->get_order_id( $order );
 			$items = array_filter( $order->get_items(), array( $this, 'filter_items_needing_shipping' ) );
 			$items_count = array_reduce( $items, array( $this, 'reducer_items_quantity' ), 0 );
-
 			$payload = array(
-				'order'             => $order->get_data(),
+				'order'             => $connect_order_presenter->get_order_for_api( $order ),
```

Comment:
The data here should match the format in https://github.com/Automattic/woocommerce-services/blob/develop/tests/e2e-tests/fixtures/orders.json